acceptance/tests/environment/directory_environment_with_access_problems.rb

```diff
@@ -0,0 +1,42 @@
+test_name 'Ensure that agent receives proper error message when environment is inaccessible'
+require 'puppet/acceptance/classifier_utils'
+extend Puppet::Acceptance::ClassifierUtils
+
```

Comment:
Can you add a `tag 'server'` here so that this runs in the pipelines testing master functionality vs those testing agent functionality?

see https://github.com/puppetlabs/puppet/blob/master/acceptance/tests/environment/use_enc_environment.rb#L5-L7 for an example

Reply:
filed https://tickets.puppetlabs.com/browse/PUP-8069 to track the tagging so we don't lose it